modules/google-analytics/wp-google-analytics.php

```diff
@@ -0,0 +1,615 @@
+<?php
+/**
+ * Plugin Name: WP Google Analytics
```

Comment:
We should remove this header completely, leaving in the copyright note below it.